packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,165 @@
+1. **Label:** Labels ChatBot messages as "AI."
+1. **Timestamp:** The relative or absolute time that a message was sent.
+1. **Quick responses:** Programmable, clickable actions that allow users to quickly answer questions from the ChatBot.
+
+### Footnotes 
+
+The footnote provides a persistent space to display messaging about your product's Terms and Conditions, which focus on the rules of using the service, and Privacy Policy, which focuses on the handling of personal data.
+
+Though footnotes are not required, they are highly recommended to ensure legal compliance, establish user trust, and clearly define the usage guidelines and data handling practices. 
+
+When users select the footnote, you can display a popover that provides more information than would fit in the footnote:
+
+<div class="ws-docs-content-img">
+![Selected footnote, with an informative popover.](./img/footnote.svg)
+</div>
+
+## Usage
+
+When ChatBots are designed to meet the needs of your users, they can improve the overall UX of your product by offering convenient, efficient, and persistent support. When your ChatBot cannot find an answer for your users, you must provide them with a method to contact human support.
+
+Before building a ChatBot, make sure that you have justified it as an appropriate solution by asking yourself these questions:
+- What are the users’ goals?
+- How in-depth is the assistance the user will need?
+- Does human assistance better serve your users?
+- How is a ChatBot superior to online documentation, contextual support or wizards?
+- What data sources or abilities can this ChatBot leverage to assist your users?
+
+Do not create a ChatBot simply for the sake of novelty.
+
+### When to use a ChatBot 
+
+Use a ChatBot to offer your users on-demand help at any time, including:
+- Technical support and troubleshooting.
+- Product information and documentation.
+- Sales and product recommendations.
+- Training and onboarding.
+- System monitoring and alerts.
+- Community engagement .
+- Feedback collection and surveys.
+
+### When not to use a ChatBot
+
+Do not use a ChatBot when: 
+- A task could be accomplished more efficiently through the UI. 
+- A process is very complex or could take a long time.
+- A real human is needed for sensitive or emotional topics.
+
+## Behavior 
+
+### Accessing a ChatBot
+
+Users can enter a conversation with a ChatBot by clicking on the toggle. Once the ChatBot window opens, the toggle will change to display an "angle down" icon to indicate that clicking the toggle again will minimize the ChatBot. Users can select the toggle at any point in their journey to open and close the ChatBot as needed.
+
+<div class="ws-docs-content-img">
+![2 toggles with open and closed icons.](./img/chatbot-toggle.svg)
+</div>
+
+When there is an unread message from the ChatBot, a notification badge should be placed on the toggle.
```

Comment:
Would we also want to discuss adding your own image?

Reply:
I added a small note to the annotation at the top, but can expand further if that's helpful!